src/renderer/src/components/mobile/MobileBrandIcons.tsx

```
@@ -1,15 +1,19 @@
 // Why: these are exact filled brand paths, not generic line approximations.
-export function IosBrandIcon(): React.JSX.Element {
+type MobileBrandIconProps = {
+  className?: string
+}
+
+export function IosBrandIcon({ className }: MobileBrandIconProps = {}): React.JSX.Element {
   return (
-    <svg className="mp-platform-brand-icon" viewBox="0 0 24 24" aria-hidden>
+    <svg className={className ?? 'mp-platform-brand-icon'} viewBox="0 0 24 24" aria-hidden>
```

Biome (2.5.0)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)